exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapDrillTable.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I think a link to a JIRA can be added. All needed information about already listed there.